.github/workflows/edFi.common manual.yml

```diff
@@ -0,0 +1,48 @@
+name: EdFi.Common Manually triggered build
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  BUILD_INCREMENTER: "10"
```

Comment:
I believe this one will need to be 11 since there is currently a 5.4.11 out in TeamCity right now, so we'd need this to be 11 so that 11 + first build 1 would be 12.

Reply:
yes updated . Thanks